include/miopen/miopen.h

```diff
@@ -68,6 +68,7 @@
  * @defgroup argmax
  * @defgroup groupnorm
  * @defgroup cat
+ * @defgroup pad_constant
```

Comment:
Check https://github.com/moreh-dev/MIOpen/pull/7/files#r1568875046

Reply:
I've changed the group but I expect to have to merge the directories as well later